src/getBullsAndCows.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+function getBullsAndCows(secret, guess) {
+  const secretArr = secret.toString().split('');
+  const guessArr = guess.toString().split('');
+  let bulls = 0;
+  let cows = 0;
+
+  for (let i = 0; i < secretArr.length; i++) {
```

Comment:
Consider using 'forEach' instead of a traditional loop for better readability and functional programming.